doc/X-OR_tutorial/X-OR_tutorial.rst

```diff
@@ -301,9 +301,9 @@ Just take a look at the way we handle types if you are not familiar with LLVM ty
     }
 
 
-The most interesting part in rewriteAsBase2 is the use of `APInt` to hold the `Base^{OriginalNbBit - 1}` value.
+The most interesting part in rewriteAsBase2 is the use of ``APInt`` to hold the :math:`base^{L - 1}` value.
```

Comment:
rewriteAsBase2 doesn't seem a good name for the function, since it *does not* exactly rewrites in base 2 (there's a modulo 2 on the bits before)